test/common/common.go

```diff
@@ -258,3 +258,19 @@ func DoRootComplianceTest(hub dynamic.Interface, policyName string, compliance p
 		1,
 	).Should(gomega.Equal(compliance))
 }
+
+func OutputDebugInfo(testName string, additionalResources []string) {
```

Comment:
Nit: could be variadic for the additionalResources - currently to not get any additional resources, you need to pass `nil`. But this is totally not necessary right now.

Reply:
That's a nice suggestion!